src/commands/game_commands/play.ts

```diff
@@ -23,22 +31,95 @@ export default class PlayCommand implements BaseCommand {
             }
             if (!gameSessions[message.guildID]) {
                 const textChannel = message.channel;
-                const gameSession = new GameSession(textChannel, voiceChannel, message.author);
+                const eliminationMode = parsedMessage.components.length >= 1 && parsedMessage.components[0].toLowerCase() === "elimination";
+                const gameOwner = message.author;
+                const lives = parsedMessage.components.length > 1 ? parseInt(parsedMessage.components[1], 10) : DEFAULT_LIVES;
+                const gameSession = new GameSession(textChannel, voiceChannel, gameOwner, eliminationMode, lives);
                 gameSessions[message.guildID] = gameSession;
-                await sendInfoMessage(message, `Game starting in #${textChannel.name} in 🔊 ${voiceChannel.name}`, "Listen to the song and type your guess!");
-                logger.info(`${getDebugContext(message)} | Game session starting`);
-                gameSession.startRound(guildPreference, message);
+
+                const lifeOrLives = lives === 1 ? "life" : "lives";
+                const joinReaction = "✋";
+                const startReaction = "🎵";
+                const gameInstructions = eliminationMode ? `Click on ${joinReaction} to join the game. ${bold(getUserIdentifier(gameOwner))} needs to press ${startReaction} to start it! Everyone begins with ${lives} ${lifeOrLives}.` : "Listen to the song and type your guess!";
+                const startTitle = `Game starting in #${textChannel.name} in 🔊 ${voiceChannel.name}`;
+                const startMessage = await sendInfoMessage(message, "Use the reactions to setup the game", gameInstructions);
+                if (eliminationMode) {
+                    startMessage.addReaction(joinReaction);
+                    startMessage.addReaction(startReaction);
+                    state.client.on("messageReactionAdd", (msg, emoji, reactor) => {
+                        if (msg.id !== startMessage.id || emoji.name !== startReaction || reactor !== gameOwner.id) {
+                            return;
+                        }
+                        const participants: { [userID: string]: {tag: string, avatar: string} } = {};
+
+                        // When startReaction pressed, get users who reacted to joinReaction and set as participants
+                        Promise.resolve(state.client.getMessageReaction(textChannel.id, msg.id, joinReaction))
+                            .then((reactors) => {
+                                reactors.forEach((player) => {
+                                    if (player.id === state.client.user.id) {
+                                        return;
+                                    }
+                                    participants[player.id] = { tag: getUserIdentifier(player), avatar: player.avatarURL };
+                                });
+                                if (Object.keys(participants).length === 0) return;
```

Comment:
Should there be a warning message in this case?

Reply:
As it is now, it'll ignore the reaction and let the user retry as many times as needed before someone has joined as a participant. Spamming this reaction could lead to multiple warning messages being sent, which isn't ideal. However, I could limit this to only once if you'd prefer a warning message.